### PR TITLE
format-regression-test-logs: implement diff instead of calling "diff" command

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: groonga-query-log.gemspec [hunk 1]
```diff
@@ -56,6 +56,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency("groonga-client")
   spec.add_runtime_dependency("groonga-log", ">= 0.1.2")
   spec.add_runtime_dependency("groonga-command", ">= 1.4.2")
+  spec.add_runtime_dependency("diff-lcs")
```
Comment (kou) [inline groonga-query-log.gemspec]: Could you sort alphabetically?

File: test/command/test-format-regression-test-logs.rb [hunk 2]
```diff
@@ -56,9 +56,8 @@ def test_command_format
 Name: select
 Arguments:
   table: Logs
---- old
-+++ new
```
Comment (kou) [inline test/command/test-format-regression-test-logs.rb]: Could you add them?

File: test/command/test-format-regression-test-logs.rb [hunk 3]
```diff
@@ -56,9 +56,8 @@ def test_command_format
...
-@@ -1,4 +1,4 @@
+
```
Comment (kou) [inline test/command/test-format-regression-test-logs.rb]: This empty line is needless.

File: test/command/test-run-regression-test.rb [hunk 4]
```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```
Comment (kou) [inline test/command/test-run-regression-test.rb]: Could you remove this?

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

**Pull request merged**